packages/EasyStandard/src/Rector/PhpDocCommentRector.php

```diff
@@ -0,0 +1,246 @@
+
+                continue;
+            }
+
+            if ($phpDocChildNode instanceof AttributeAwarePhpDocTagNode) {
+                $this->checkTagNode($phpDocChildNode);
+
+                continue;
+            }
+        }
+
+        $this->isMultilineTagNode = false;
+    }
+
+    private function checkTagNode(AttributeAwarePhpDocTagNode $attributeAwarePhpDocTagNode): void
+    {
+        if ($attributeAwarePhpDocTagNode->value instanceof AttributeAwareGenericTagValueNode) {
+            $this->checkGenericTagValueNode($attributeAwarePhpDocTagNode);
+        }
+
+        if ($attributeAwarePhpDocTagNode->value instanceof AttributeAwareVarTagValueNode) {
+            $this->checkVarTagValueNode($attributeAwarePhpDocTagNode);
+        }
+    }
+
+    private function checkTextNode(AttributeAwarePhpDocTextNode $attributeAwarePhpDocTextNode): void
+    {
+        if ($this->isLineStartsWithAllowedAnnotation($attributeAwarePhpDocTextNode->text)) {
+            $this->isMultilineTagNode = true;
+
+            return;
+        }
+
+        if ($attributeAwarePhpDocTextNode->text === '' || Strings::endsWith($attributeAwarePhpDocTextNode->text, ')')) {
+            $this->isMultilineTagNode = false;
+
+            return;
+        }
+
+        $text = (array)\explode(PHP_EOL, $attributeAwarePhpDocTextNode->text);
+        $firstKey = array_key_first($text);
+        $lastKey = array_key_last($text);
+
+        foreach ($text as $index => $value) {
+            $text[$index] = Strings::trim($value);
+        }
+
+        $text[$firstKey] = Strings::firstUpper($text[$firstKey]);
+
+        if ($this->isLineEndingWithAllowed($text[$lastKey]) === false) {
+            $text[$lastKey] .= '.';
+        }
+
+        $attributeAwarePhpDocTextNode->text = \implode(PHP_EOL, $text);
+        $attributeAwarePhpDocTextNode->setAttribute('original_content', '');
+    }
+
+    private function checkVarTagValueNode(AttributeAwarePhpDocTagNode $attributeAwarePhpDocTagNode): void
+    {
+        if ($this->isMultilineTagNode) {
+            return;
+        }
+
+        /** @var AttributeAwareVarTagValueNode $varTagValueNode */
+        $varTagValueNode = $attributeAwarePhpDocTagNode->value;
+
+        if ($varTagValueNode->description === '' || $varTagValueNode->variableName === '') {
+            return;
+        }
+
+        $varTagValueNode->description = Strings::firstLower(\trim($varTagValueNode->description));
+
+        if ($this->isLineEndingWithAllowed($varTagValueNode->description)) {
+            $varTagValueNode->description = Strings::substring($varTagValueNode->description, 0, -1);
+        }
+    }
+
+    private function isLineEndingWithAllowed(string $docLineContent): bool
+    {
+        $lastCharacter = Strings::substring($docLineContent, -1);
+
+        return \in_array($lastCharacter, $this->allowedEnd, true);
+    }
+
+    private function isLineStartsWithAllowedAnnotation(string $docLineContent): bool
```

Comment:
"Is starts" in incorrect. The method should be named either `isLineStartingWithAllowedAnnotation` or `doesLineStartWithAllowedAnnotation`

Reply:
Reworked.